src/modules/feature-modules/admin/pages/service-users/service-user-info.component.html

```diff
@@ -29,7 +29,7 @@ <h2 class="nhsuk-heading-m">Innovations</h2>
     </section>
   </div>
 
-  <div *ngIf="userInfoType === 'ACCESSOR'" >
+  <div *ngIf="userInfoType === userRoleEnum.ACCESSOR || userInfoType === userRoleEnum.QUALIFYING_ACCESSOR" >
```

Comment:
The more I see the PRs the more I feel we need an helper for this. 

(example code only)
```
// authentication.store.ts
checkAccessorType(userType: UserRoleEnum): boolean {
   return [UserRoleEnum.ACCESSOR, UserRoleEnum.QUALIFYING_ACCESSOR].includes(userType);
}
```

Then we would just call:

```
<... *ngIf="stores.authentication.checkAccessorType(userInfoType) ...>
```